pupa/settings.py

```diff
@@ -72,3 +73,10 @@
 
 
 DATABASES = {'default': dj_database_url.parse(DATABASE_URL)}
+
+
+elasticsearch = None
```

Comment:
I'm down to put it together, just give me an idea of what else should be moved to `core`. Or I can just roughly mimic the `billy.core`.